documentation/modules/security/proc-recovering-expired-ca-certs.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-recovering-expired-ca-certs-{context}']
+
+= Manually recovering from expired Cluster Operator-managed CA certificates
+
+[role="_abstract"]
+Cluster and clients CA certificates are automatically renewed by the Cluster Operator at the start of their renewal periods. 
+However, under certain circumstances, the renewal process may fail due to operational issues or disruptions. 
+In such an event, it is important to recover your deployment from using expired certificates to ensure secure communication. 
+
+You can recover from expired cluster and clients CA certificates. 
+The process involves deleting the secrets containing the expired certificates so that new ones are generated by the Cluster Operator. 
+For more information on the secrets managed in Strimzi, see xref:con-certificates-str[].
+
+To perform a recovery, follow the steps outlined in this procedure in the order given. 
+
+NOTE: If you are using your own CA certificates, follow the procedure for xref:renewing-your-own-ca-certificates-{context}[renewing your own CA certificates].
```

Comment:
Actually, if a custom CA expires, the process will be similar. But the user has to renew the CA themselves.

Reply:
changed to:

> NOTE: If you are using your own CA certificates and they expire, the process is similar, but you need to xref:xref:renewing-your-own-ca-certificates-{context}[renew the CA certificates] rather than use new certificates generated by the Cluster Operator.